### PR TITLE
Add csv option to count overlap.py

### PR DESCRIPTION
**Pull request by drtamermansour**
I have added the CSV option to the count-overlap.py and I added a test for this in test_scripts.py

### CODE REVIEW

**Comment by drtamermansour** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by drtamermansour** [comment]
ready for review

**Comment by drtamermansour** [comment]
@ctb  would you please check the header names

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,17 @@
+2015-02-26  Tamer A. Mansour  <drtamermansour@gmail.com>
```
Comment (ctb) [inline ChangeLog]: You can combine these ChangeLog entries.

**Comment by ctb** [comment]
The header names look good -- thanks!

**Comment by drtamermansour** [comment]
Ready for review

File: scripts/count-overlap.py [hunk 2]
```diff
@@ -81,8 +89,10 @@ def main():
     output.write(printout1)
 
     for i in range(100):
-        to_print = str(list_curve[100 + i]) + ' ' + str(list_curve[i]) + '\n'
-        f_curve_obj.write(to_print)
+        if args.csv:
+            f_curve_obj_csv.writerow([list_curve[100 + i], list_curve[i]])
+        else:
+            print >> f_curve_obj, list_curve[100 + i], list_curve[i]
```
Comment (ctb) [inline scripts/count-overlap.py]: There's no test coverage for this line... could you add, please?

**Comment by drtamermansour** [comment]
It was already covered, the problem was the name of the test function. It should be ok now

**Comment by drtamermansour** [comment]
ready for review

**Comment by ctb** [comment]
Merged, tnx!

**Pull request merged**